tests/convention_test.py

```diff
@@ -0,0 +1,74 @@
+"""Unit tests for the metadata block in project README files."""
+
+from pathlib import Path
+
+import update_projects_table
+
+import metadata_definitions
+import pytest
+
+
+ROOT_PATH = Path(__file__).parent.parent
+
+TEST_CATEGORIES = [
+    (Path(ROOT_PATH / "devops"), "DevOps"),
+    (Path(ROOT_PATH / "reliability"), "Reliability"),
+    (Path(ROOT_PATH / "samples"), "Samples"),
+]
+
+METADATA_VALID = [
+    (metadata_definitions.CATEGORIES, "categories"),
+    (metadata_definitions.INTEGRATIONS, "integrations"),
+]
+
+
+def test_metadata_block_exists():
+    """Checks for required metadata in README files."""
+    for readme_file in sorted(Path(ROOT_PATH).rglob("README.md")):
+        metadata = update_projects_table.extract_metadata(readme_file)
+
+        assert metadata, f"Metadata block is missing in the README file: {readme_file}"
+        assert all(key in metadata for key in metadata_definitions.METADATA), (
```

Comment:
The first assert is wrong, as noted above.

As for asserts 2-4:
1. Should be in separate test functions because they're not related to each other; we don't have an error in assert 2 to hide additional errors in asserts 3 and 4
2. The relative path of a project is more useful than its title, to know where the fix should be, also it's easier for the eye if there's a visual separation, like:
   ```py
   error = "blah blah blah: /dir/subdir/README.md"
   ```